src/vma/dev/cq_mgr.h

```diff
@@ -305,4 +304,34 @@ class cq_mgr
 // Since we have a single TX CQ comp channel for all cq_mgr's, it might not be the active_cq object
 cq_mgr* get_cq_mgr_from_cq_event(struct ibv_comp_channel* p_cq_channel);
 
+#ifdef HAVE_INFINIBAND_MLX5_HW_H
+
+class cq_mgr_mlx5: public cq_mgr
```

Comment:
Done for qp's - still have to show me what the problem is with cq's. `inline` stuff, I know, but lets go over it...